Makefile

```
@@ -0,0 +1,13 @@
+dev_compose_file := docker-compose.dev.yml
+test_compose_file := docker-compose.test.yml
+application_container_test := application_test
+
+.PHONY: dev-start dev-stop test
+
+dev-start:
+	docker compose -f $(dev_compose_file) up -d
+	yarn migrate:dev
+	yarn dev
+
+dev-stop:
+	docker compose -f $(dev_compose_file) down -v
```

For local dev, there would be no need to remove each time, so the following would suffice:

```suggestion
	docker compose -f $(dev_compose_file) stop -v
```

---

Fixed